Router.svelte

```diff
@@ -173,25 +173,37 @@ export function replace(location) {
  *
  * @param {HTMLElement} node - The target node (automatically set by Svelte). Must be an anchor tag (`<a>`) with a href attribute starting in `/`
  */
-export function link(node) {
+export function link(node, h) {
```

Comment:
Would be great if you could find a more descriptive name for the parameter than `h`